cmake/modules/SIMDExt.cmake

```diff
@@ -1,66 +1,60 @@
 # detect SIMD extentions
 #
-# ARM_NEON_FLAGS
-#
 # HAVE_ARMV8_CRC
```

Comment:
the flags are used primarily in gf-complete and zlib. they were based on #define that originated in gf-complete and are non-standard. I'll attempt to split the commit as you suggest and add some verbiage.